### ISSUE TITLE
speedup SlickProjectionSpec

### ISSUE THREAD

**Issue by patriknw** [akka/akka-projection#64]
Must be possible to speedup

```
[info] SlickProjectionSpec:
[info] A Slick projection
[info] - must persist projection and offset in same the same write operation (transactional) (2 seconds, 550 milliseconds)
[info] - must restart from previous offset - fail with DBIOAction.failed (6 seconds, 200 milliseconds)
[info] - must restart from previous offset - fail with throwing an exception (6 seconds, 206 milliseconds)
[info] - must restart from previous offset - fail with bad insert on user code (6 seconds, 192 milliseconds)
```

Corresponding CassandraProjectionSpec:
```
[info] CassandraProjectionSpec:
[info] A Cassandra projection
[info] - must persist projection and offset (662 milliseconds)
[info] - must restart from previous offset - handler throwing an exception (3 seconds, 745 milliseconds)
[info] - must save offset after number of elements (919 milliseconds)
[info] - must save offset after idle duration (2 seconds, 296 milliseconds)
```

**Comment by renatocaval** [comment]
No only that, I think it takes very long to bootstrap the tests. 

We are not yet using a connection pool. That should improve timing between the tests, but not the bootstrap.

**Comment by renatocaval** [comment]
Fixed in #64
